tests/unittests/OperatorTest.cpp

```diff
@@ -180,6 +180,26 @@ TEST_P(Operator, matmul) {
   EXPECT_NEAR(H.at({2, 0}), 95, 0.001);
 }
 
+TEST_P(InterpOnly, FP16Matmul) {
```

Comment:
nit: comment for what is being tested? It is kind of obvious from the test name...

Reply:
Done!